1_fetch/src/get_nlcd_LC.R

```diff
@@ -0,0 +1,90 @@
+download_NHD_NLCD_data <- function(sb_id,
+                                 out_path = '1_fetch/src',
+                                 downloaded_data_folder_name = 'NA',
+                                 create_LandCover_folder = T, overwrite_download = T){
+  
+  #' @description download Land Cover data to the repo's fetch/src folder.
```

Comment:
Can you change this to go to '1_fetch/out'? You'll have to add a dir.create for that directory.